etc/run_echo_server.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd tests/goutils/rpc/examples/echo || exit 1
+
+go run server/cmd/main.go -instance-name="localhost:$1" &>/dev/null
```

Comment:
I disabled output from the server to make the output less noisy

Reply:
Do you think it makes sense to add an option for more verbose output, if e.g. someone wanted to run this locally?

Reply:
Just gonna keep it verbose for now for sake of keeping the bash scripts as light as possible: https://github.com/viamrobotics/rust-utils/pull/39/commits/dcf1b587778a0d4f9a4e21d1f029e105fe44082e